pkg/tasks/task.go

```diff
@@ -41,22 +41,23 @@ type Task struct {
 }
 
 // Run runs a task
-func (t *Task) Run(ctx *state.State) error {
+func (t *Task) Run(s *state.State) error {
 	if t.Retries == 0 {
-		t.Retries = 1
+		t.Retries = 3
 	}
+
 	backoff := defaultRetryBackoff(t.Retries)
 
 	var lastError error
 	err := wait.ExponentialBackoff(backoff, func() (bool, error) {
 		if lastError != nil {
-			ctx.Logger.Warn("Retrying task…")
+			s.Logger.Warn("Retrying task…")
 		}
-		lastError = t.Fn(ctx)
+		lastError = t.Fn(s)
 		if lastError != nil {
-			ctx.Logger.Warn("Task failed…")
-			if ctx.Verbose {
-				ctx.Logger.Warnf("error was: %s", lastError)
+			s.Logger.Warn("Task failed…")
+			if s.Verbose {
+				s.Logger.Warnf("error was: %s", lastError)
 			}
```

Comment:
```suggestion
			s.Logger.Warnf("error was: %s", lastError)
```

I propose that we just print the error when the task fails regardless of is verbose enabled or not. In some cases, such as `kubeadm init` or `kubeadm join`, the error can be different between retries and that can be confusing. Printing error each time it happens should make the user experience a little bit better.

Reply:
👍 sounds good